api/base/filters.py

```diff
@@ -215,6 +218,8 @@ def convert_value(self, value, field):
                 )
         elif isinstance(field, self.LIST_FIELDS):
             return value
+        elif isinstance(field, RelationshipField):
```

Comment:
This will conflict with a [PR](https://github.com/CenterForOpenScience/osf.io/pull/4693/files#diff-8a17f6dba621ae1374cefd4514bfc739R219) that @samanehsan has open. No action required at this time, just a heads up.